build.gradle

```diff
@@ -218,19 +214,15 @@ task copyToLibs(type: Copy) {
 }
 
 task copyToRoot(type: Copy, dependsOn: [copyToLibs, jar, converterJar]) {
-	outputs.files.setFrom(file("$buildDir/libs/pcgen-${version}.jar"))
```

Comment:
I've tried removing this line and I see it fail in 2.2 and 2.10. Again I think it is one of those unusual workarounds I needed to do to get it working correctly.